src/containers/MyNdla/components/NotficationPopover.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2022-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { colors, spacing } from '@ndla/core';
+import { SafeLinkButton } from '@ndla/safelink';
+import styled from '@emotion/styled';
+import { Root, Trigger, Portal, Content, Arrow } from '@radix-ui/react-popover';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import NotificationList from './NotificationList';
+import { toAllNotifications } from '../../../routeHelpers';
+import NotificationBellButton from './NotificationButton';
+import { useArenaNotifications } from '../arenaQueries';
+
+const StyledContent = styled(Content)`
+  background-color: ${colors.background.default};
+  box-shadow: 0 0 ${spacing.nsmall} #ccc;
```

Comment:
Skulle fargen vore definert i colors?

Reply:
jeg er usikker. Jeg ser nå at Hedvig har brukt fading, jeg brukte den fargen der i god tro da den gråa ut å lik ut som designet. Jeg skal se om jeg får til fading med variablene våre!

Reply:
Fant på en lur løsning. Ved å legge på en heksadecimal bak så kan man styre opacity på fargene. 
[Stackoverflow for the rescue](https://stackoverflow.com/questions/13248289/css-setting-background-opacity-without-rgba)